.circleci/config.yml

```diff
@@ -680,39 +680,44 @@ workflows:
           transport-layer: "udp"
           environment-type: "production"
           requires:
-            - install-raiden-linux-3.7
```

Comment:
Why is the requirement `install-raiden-linux-3.7` removed?

Reply:
It's transitively required through the linting job.